x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_details/monitor_errors/monitor_errors.tsx

```diff
@@ -80,7 +80,7 @@ const EmptyErrors = () => {
 };
 
 const KEEP_CALM = i18n.translate('xpack.synthetics.errors.keepCalm', {
-  defaultMessage: 'Keep calm and carry on.',
+  defaultMessage: 'Keep calm.',
```

Comment:
@florent-leborgne Can you help us out with good empty state content for monitor errors?

Reply:
```suggestion
  defaultMessage: 'This monitor ran successfully during the selected period. Increase the time range to check for older errors.',
```
Would something like this work in all cases?

Reply:
Yes, that LGTM!